plugins/hipchat.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * The Hipchat integration plugin
+ * @module Hipchat
+ */
+"use strict";
+
+var HipchatApi = require('hipchatter');
+
+/**
+ * @class Hipchat
+ * @param config {Object} The plugins configs
+ * @param mergeatron {Mergeatron} An instance of the main Mergeatron object
+ * @constructor
+ */
+var Hipchat = function(config, mergeatron) {
+	this.config = config;
+	this.mergeatron = mergeatron;
+	this.api = new HipchatApi(this.config.token);
+};
+
+/**
+ * Translates a github user to a hipchat user
+ *
+ * @method getUser
+ * @param githubUser {String}
+ * @return {String}
+ */
+Hipchat.prototype.getUser = function(githubUser) {
+	if (this.config.github_to_hipchat_users && this.config.github_to_hipchat_users[githubUser]) {
+		return this.config.github_to_hipchat_users[githubUser];
+	}
+	return githubUser;
+};
+
+/**
+ * Post a message to hipchat
+ *
+ * @method sendUserMessage
+ * @param user {String}
+ * @param msg {String}
+ */
+Hipchat.prototype.sendUserMessage = function(user, msg) {
+	this.api.send_private_message('@' + user, {
+		message_format: 'text',
+		message: msg
+	}, function (err) {
+		if (err) {
+			this.mergeatron.log.error(err);
+		}
+	});
+};
+
+
+exports.init = function(config, mergeatron) {
+	var hipchat = new Hipchat(config, mergeatron);
+
+	mergeatron.on('build.started', function(job, pull, build_url) {
+		var user = hipchat.getUser(pull.user.login);
```

Comment:
was thinking about refactoring the duplicated message code here, but I thought i'd leave it in case we decide to change messages on a per event basis